Exiled.API/Extensions/ItemExtensions.cs

```diff
@@ -197,12 +197,7 @@ public static byte GetMaxAmmo(this ItemType item)
         /// </summary>
         /// <param name="items">The items to convert.</param>
         /// <returns>A new <see cref="List{T}"/> of <see cref="ItemType"/>s.</returns>
-        public static IEnumerable<ItemType> GetItemTypes(this IEnumerable<Item> items)
-        {
-            Item[] arr = items.ToArray();
-            for (int i = 0; i < arr.Length; i++)
-                yield return arr[i].Type;
-        }
+        public static IEnumerable<ItemType> GetItemTypes(this IEnumerable<Item> items) => items.Select(item => item.Type);
```

Comment:
i think it's useless method.

Reply:
Maybe

Reply:
Would not
```
foreach (Item item in items)
    yield return item.Type;``` not be more time efficient than linq, here?

Reply:
Linq should do the same thing, but with predicates. It won't give such performance boost